### PR TITLE
Adds LARPing.

### PR DESCRIPTION
**Pull request by lordpidey**
:cl:
add: The wizard federation has teamed up with various LARP groups to invent a new type of spell.
/:cl:

So, this is pretty much just a meme PR, it gives a new wizard spell with a super low cooldown that summons a packet that stuns and stamina damages on throw hit, in addition to making the wizard run around yelling "LIGHTNING BOLT!!" all the time.

### CODE REVIEW

**Comment by bunny232** [comment]
yes MAGIC MISSILE MAGIC MISSILE MAGIC MISSILE

**Comment by XDTM** [comment]
Since you're supporting illusory electrocution you might want to use that in the electrocution hallucination instead of its custom code

**Comment by lordpidey** [comment]
You mean the one on [airlock.dm line 198](https://github.com/tgstation/tgstation/blob/master/code/game/machinery/doors/airlock.dm#L198)?  Because that's coded directly into the door method.  Also, if I were to copy/paste what's written there, other people wouldn't see the "electrocution".

**Comment by YakumoChen** [comment]
Does using the spell prime your throw like priming a grenade does?

**Comment by Iamgoofball** [comment]
BRICKSLEY!

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -169,15 +169,18 @@ mob/living/carbon/bullet_act(obj/item/projectile/P, def_zone)
 		O.emp_act(severity)
 	..()
 
-/mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+/mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, safety = 0, override = 0, tesla_shock = 0, illusion = 0)
```
Comment (Jordie0608) [inline code/modules/mob/living/carbon/carbon_defense.dm]: You don't need to change these children, they inherit `illusion = 0` from the parent proc definition arguments.

File: code/modules/spells/spell_types/conjure.dm [hunk 2]
```diff
@@ -65,16 +65,19 @@
...
 /obj/effect/proc_holder/spell/targeted/conjure_item/cast(list/targets, mob/user = usr)
-	if (item)
+	if (item && !qdeleted(item))
 		qdel(item)
 		item = null
 	else
 		for(var/mob/living/carbon/C in targets)
 			if(C.drop_item())
-				item = new item_type
+				item = make_item()
```
Comment (Jordie0608) [inline code/modules/spells/spell_types/conjure.dm]: Why make this a proc?

Comment (lordpidey) [inline code/modules/spells/spell_types/conjure.dm]: Originally, I was going to make it have a variety of "spell packets" at random, but then decided on just one.

**Comment by lordpidey** [comment]
@YakumoChen, good idea, I've put that in.

File: code/modules/mob/living/carbon/carbon_defense.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon_defense.dm]: It's bad practice not to change the children if you change the arguments

Comment (lordpidey) [inline code/modules/mob/living/carbon/carbon_defense.dm]: I've tested it, and it seems to work just fine on monkeys/humans.  What do you see that's wrong, is there an edge case I've missed?

Comment (optimumtact) [inline code/modules/mob/living/carbon/carbon_defense.dm]: no I'm just mad at how byond implements this, dont' wory

**Pull request merged**